### PR TITLE
DCS-1280 integration tests for prison transfer

### PR DESCRIPTION
**Pull request by GurnankCheema**
Note the text content of pages has been tested in the unit tests already

### CODE REVIEW

File: server/routes/confirmTransferAddedToRollController.test.ts [hunk 1]
```diff
@@ -2,33 +2,22 @@ import type { Express } from 'express'
 import request from 'supertest'
 import cheerio from 'cheerio'
 import { appWithAllRoutes } from './testutils/appSetup'
-import ExpectedArrivalsService from '../services/expectedArrivalsService'
```
Comment (GurnankCheema) [inline server/routes/confirmTransferAddedToRollController.test.ts:5]: surplus - now using prisonService

File: server/routes/confirmTransferAddedToRollController.ts [hunk 2]
```diff
@@ -1,11 +1,8 @@
 import { RequestHandler } from 'express'
-import type { ExpectedArrivalsService, PrisonService } from '../services'
+import type { PrisonService } from '../services'
 
-export default class CheckTransferController {
```
Comment (GurnankCheema) [inline server/routes/confirmTransferAddedToRollController.ts:4]: class renamed

File: integration_tests/integration/confirmTransferAddedToRoll.spec.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    cy.task('stubPrison', 'MDI')
+    cy.signIn()
+  })
+
+  it('Should go to choose-prisoner page', () => {
+    const checkTransferPage = CheckTransferPage.goTo(expectedArrivals.prisonTransfer.prisonNumber)
+    checkTransferPage.addToRoll().click()
+    const confirmTransferAddedToRollPage = Page.verifyOnPage(ConfirmTransferAddedToRollPage)
+    confirmTransferAddedToRollPage.addAnotherToRoll().click()
+    Page.verifyOnPage(ChoosePrisonerPage)
```
Comment (andrewrlee) [inline integration_tests/integration/confirmTransferAddedToRoll.spec.ts:35]: we should be doing some verify calls on this and confirm arrivals - so we can ensure that WPIP is called with the expected POST. 

We do this in BVL

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**